app/controllers/authenticate_controller.rb

```diff
@@ -70,7 +70,15 @@ def authenticate(input = authenticator_input)
       authenticators: installed_authenticators,
       enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV)
     )
-    render json: authn_token
+    content_type = :json
+    if encoded_response?
+      # The response will be a base64 encoded version of the JWT.
+      Rails.logger.debug("Responding with base64 encoded JWT")
```

Comment:
can you move this message to `logs.rb` as we do with other log messages?

Reply:
@InbalZilberman @shulifink please approve message

Reply:
fine